air.toml

```diff
@@ -0,0 +1,6 @@
+[format]
+line-width = 100
+indent-width = 2
+indent-style = "space"
+line-ending = "auto"
```

Comment:
My bad, this should be `"lf"` as like `.editorconfig`.
https://github.com/eitsupi/neo-r-polars/blob/59ef78d92120f9187748b8a10f95b79ecdcef53d/.editorconfig#L4

Reply:
I opened #139